src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/draftorders/upload/UploadDraftOrdersAboutToSubmitHandler.java

```diff
@@ -86,17 +87,24 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
 
         String typeOfDraftOrder = finremCaseData.getDraftOrdersWrapper().getTypeOfDraftOrder();
         List<String> warnings = new ArrayList<>();
+        String state = caseDetails.getState() == null ? null : caseDetails.getState().getStateId();
```

Comment:
Does this mean that sometimes the state is null and sometimes it isn't?

Reply:
I'd prefer it if we removed it. I think the only time it could be null is during case creation